README.md

```diff
@@ -1,2 +1,48 @@
 # xapi-profiles
 A set of documents addressing the structure of and supporting services for xAPI Profiles.
+
+This Github repository contains the xAPI Profiles Specification. [xAPI](https://github.com/adlnet/xAPI-Spec) is a learning
+technologies interoperability specification that describes communication
+about learner activity and experiences between technologies. The specification stands as a 
```

Comment:
Change to "The xAPI Profiles Specification stands as a" because we're talking about two different specs in this section.